### PR TITLE
Restructuring website resources

### PR DESCRIPTION
**Pull request by d-werner**
- Cleaned up website resources and moved them to dedicated folder `src_website`.
- Added SCSS support

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
-  "author": "You <you@example.com>",
-  "name": "rust-webpack-template",
+  "name": "extract-my-file",
```
Comment (pacman82) [inline package.json]: beats rust-webpack-template for sure. Yet I'd like to have a name which would include shrinking just as much as expanding

File: package.json [hunk 2]
```diff
@@ -9,11 +12,16 @@
...
-    "copy-webpack-plugin": "^5.0.3",
-    "webpack": "^4.33.0",
+    "css-loader": "^3.2.1",
+    "file-loader": "^5.0.2",
+    "html-webpack-plugin": "^3.2.0",
+    "node-sass": "^4.13.0",
+    "rimraf": "^2.6.3",
+    "sass-loader": "^8.0.0",
+    "style-loader": "^1.0.1",
+    "webpack": "^4.41.2",
```
Comment (pacman82) [inline package.json]: For most of these I don't know what they are doing. We need to grab a cup of coffee together, so you can explain this to me. Until then, I'll just nod and merge.

**Comment by pacman82** [comment]
Merging this. Yet: What is SCSS?

**Comment by d-werner** [comment]
SCSS is an extended CSS language with variables, functions and hierarchical styling declarations.
It is actually just a flavor of SASS: https://sass-lang.com/

File: package.json (hunk 1 above)
Comment (d-werner) [reply package.json]: That's true. Just wanted it aligned with the current website title first. ;-)

**Pull request merged**